rainier-core/src/main/scala/com/stripe/rainier/ir/GraphViz.scala

```diff
@@ -0,0 +1,77 @@
+package com.stripe.rainier.ir
```

Comment:
this is in the IR package because I intend to add an `ir.Viz` as well that makes us of it to visualize IR trees